test/luatest_helpers/vtest.lua

```diff
@@ -741,6 +741,18 @@ local function service_wait_for_activity(service, activity, opts)
     end)
 end
 
+-- Git directory of the project and data directory of the test.
+-- Used in evolution tests to fetch old versions of vshard.
+local SOURCEDIR = fio.abspath(os.getenv('PACKPACK_GIT_SOURCEDIR') or
+                              os.getenv('SOURCEDIR'))
+if not SOURCEDIR then
+    local script_path = debug.getinfo(1).source:match("@?(.*/)")
+    script_path = fio.abspath(script_path)
+    SOURCEDIR = fio.abspath(script_path .. '/../../../')
+end
+
+local VARDIR = fio.abspath(os.getenv('VARDIR') or './var')
```

Comment:
Please describe why do you need this directory (as for SOURCEDIR). And probably why can't you use temporary directory for that purpose.

Reply:
Actually, I don't. Temporary directories will work just fine, they're unique enough to allow running test in parallel. Thank you for idea